### PR TITLE
init

### PR DESCRIPTION
**Pull request by mingshewhe**

### CODE REVIEW

**Comment by tencentblueking-adm** [comment]
[CLA assistant check](https://bk-cla.bktencent.com:443/TencentBlueKing/iam-java-sdk?pullRequest=1) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/iam-java-sdk?pullRequest=1) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/iam-java-sdk?pullRequest=1) it.</sub>

File: gradle.properties [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT
+# LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN
+# NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+# WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
```
Comment (wklken) [inline gradle.properties:25]: license 头需要改成 iam-java-sdk 的

https://github.com/TencentBlueKing/iam-python-sdk/blob/master/iam/collection.py#L4

File: src/main/java/com/tencent/bk/sdk/iam/config/IamConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package com.tencent.bk.sdk.iam.config;
```
Comment (wklken) [inline src/main/java/com/tencent/bk/sdk/iam/config/IamConfiguration.java:1]: 每个源文件都得带 license头?

File: src/main/java/com/tencent/bk/sdk/iam/config/IamConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+/**
+ * @author citruswang
```
Comment (wklken) [inline src/main/java/com/tencent/bk/sdk/iam/config/IamConfiguration.java:7]: 应该去掉`@author`这类信息; 更多的是以git blame去确定属主, 不再代码中维护; 并且这个信息相对敏感

File: src/main/java/com/tencent/bk/sdk/iam/config/IamConfiguration.java (hunk 2 above)
Comment (mingshewhe) [reply src/main/java/com/tencent/bk/sdk/iam/config/IamConfiguration.java:12]: https://github.com/TencentBlueKing/iam-java-sdk/pull/1/files#diff-71ea8eebcffbfe7281b61e73d464221960ae687e4532c3a25aa7af2890cf016a 已全部修改成iam-java-sdk对应的license

File: src/main/java/com/tencent/bk/sdk/iam/config/IamConfiguration.java (hunk 3 above)
Comment (mingshewhe) [reply src/main/java/com/tencent/bk/sdk/iam/config/IamConfiguration.java]: 已全部修复

File: README.md [hunk 4]
```diff
@@ -1 +1,70 @@
...
+
+## IAM Repos
+
+- [TencentBlueKing/bk-iam](https://github.com/TencentBlueKing/bk-iam)
+- [TencentBlueKing/bk-iam-saas](https://github.com/TencentBlueKing/bk-iam-saas)
+- [TencentBlueKing/bk-iam-search-engine](https://github.com/TencentBlueKing/bk-iam-search-engine)
+- [TencentBlueKing/bk-iam-cli](https://github.com/TencentBlueKing/bk-iam-cli)
+- [TencentBlueKing/iam-python-sdk](https://github.com/TencentBlueKing/iam-python-sdk)
+- [TencentBlueKing/iam-go-sdk](https://github.com/TencentBlueKing/iam-go-sdk)
+- [TencentBlueKing/iam-php-sdk](https://github.com/TencentBlueKing/iam-php-sdk)
```
Comment (wklken) [inline README.md:45]: 这里加下java sdk自己的链接 => 合并后其他所有相关的项目会一并更新, 让开发者能在任意iam 项目中跳转到java sdk

**Pull request merged**